dummy.test.js

```diff
@@ -1,3 +1,11 @@
-test("tests are run", () => {
-  expect(true).toBe(true);
+import countBy from "./src/countBy.js";
+
+test("countBy works", () => {
+  expect(countBy([
+     { 'user': 'barney', 'active': true },
+     { 'user': 'betty', 'active': true },
+     { 'user': 'fred', 'active': false }
+   ], value => value.active)).toBe(
+    { 'true': 2, 'false': 1 }
+  );
 });
```

Comment:
These tests are duplicated in countBy.test.js. This dummy file can probably be deleted since the test setup is now working.